vademecum_VT.md

```diff
@@ -56,7 +56,7 @@ Inoltre, A-Frame, ha diversi sottoprogetti come _A-Terrain_, per visualizzare da
 Rust è un linguaggio compilato con performance simili a quelle del C e garantisce, in aggiunta, la sicurezza del programma. È sviluppato da Mozilla ed è un componente fondamentale in Firefox Quantum.
 Rust utilizza il concetto di zero-cost abstraction, ovvero le astrazione vengono fatte in compile-time anziché in run-time, ottenendo così le massime prestazioni possibili senza sacrificare la produttività.
 Un altro concetto fondamentale di Rust è quello riguardante l'affidabilità, infatti garantisce che non si verifichi alcun errore di memoria, come i _buffer overflow_, _double free_, _null pointer_, _access_, _segfault_ e _data concurrency_.
-Grazie a efficienti C Bindings, inoltre, è possibile facilmente integrare il codice in altri linguaggi, quali NodeJS e Python.
+Grazie a efficienti C Bindings, inoltre, è possibile facilmente integrare il codice in altri linguaggi, come NodeJS e Python.
```

Comment:
non itnegrare codice ma progetti

Reply:
quindi "integrare progetti in altri linguaggi, come NodeJS e Python."?

Reply:
si esatto